docs/topics/api/hero.rst

```diff
@@ -46,3 +48,23 @@ small number of shelves this endpoint is not paginated.
     :>json string|null results[].description: The description for this item, if any. May contain some HTML tags.
     :>json object results[].addon: The :ref:`add-on <addon-detail-object>` for this item if the addon is hosted on AMO. Either this field or ``external`` will be present.  Only a subset of fields are present: ``id``, ``authors``, ``average_daily_users``, ``current_version`` (with only the ``id``, ``compatibility``, ``is_strict_compatibility_enabled`` and ``files`` fields present), ``guid``, ``icon_url``, ``name``, ``ratings``, ``previews``, ``slug``, ``theme_data``, ``type`` and ``url``.
     :>json object results[].external: The :ref:`add-on <addon-detail-object>` for this item if the addon is externally hosted. Either this field or ``addon`` will be present.  Only a subset of fields are present: ``id``, ``guid``, ``homepage``, ``name`` and ``type``.
+
+
+----------------------
+Secondary Hero Shelves
+----------------------
+
+.. _secondary-hero-shelf:
+
+This endpoint returns all enabled secondary hero shelves.  As there will only ever be a
+small number of shelves - and likely only one - this endpoint is not paginated.
+
+
+.. http:get:: /api/v4/hero/secondary/
+
+    :>json array results: The array containing the results for this query.
+    :>json string results[].headline: The headline for this item. May contain some HTML tags.
+    :>json string results[].description: The description for this item. May contain some HTML tags.
```

Comment:
I wonder if we should mention if they're escaped or not here?

Reply:
I copied this line from the discovery api documentation (and the primary shelf documentation above).  Currently it's just a char field that could contain literally anything entered in the admin, and when #11989 is implemented it'll be appropriately escaped as per the discovery api/model.  I don't know what is appropriate documentation for this at this point.  (Or even if frontend would expect html or text only for these fields until Bob is back)

Reply:
I decided to leave out any mention of html and escaping because it's not clear what the implementation will be yet.  If there ends up being html in #11989 I'll add docs then (and tests)